lib/content-services/src/lib/document-list/components/document-list.component.spec.ts

```diff
@@ -132,57 +132,39 @@ describe('DocumentList', () => {
             });
         };
 
-        it('should load -trashcan- preset', async () => {
-            documentList.currentFolderId = '-trashcan-';
-
-            fixture.detectChanges();
-            await fixture.whenStable();
-
+        it('should load -trashcan- preset', () => {
+            documentList.presetColumn = '-trashcan-';
+            documentList.ngAfterContentInit();
```

Comment:
if you need this one instead of old code, something is not working correctly. normally you don't need touching ngEvents